packages/ic-response-verification/src/hash/response_hash.rs

```diff
@@ -120,6 +120,7 @@ pub fn response_hash(
     response: &Response,
     response_certification: &ResponseCertification,
 ) -> Sha256Digest {
+    // lower case the headers here
```

Comment:
is this a todo?

Reply:
Thanks for spotting that. I meant to remove this. it's done now and I created a ticket to follow up on that instead.